cpp/src/aztec/proof_system/flavor/flavor.hpp

```diff
@@ -87,10 +88,18 @@ struct StandardHonk {
         // Round 3
         manifest_rounds.emplace_back(transcript::Manifest::RoundManifest(
             { { .name = "Z_PERM", .num_bytes = g1_size, .derived_by_verifier = false } },
+            /* challenge_name = */ "zeta",
+            /* num_challenges_in = */ 1)
+        ); 
+
+        // Round 4
+        manifest_rounds.emplace_back(transcript::Manifest::RoundManifest(
+            { { .name = "POW_ZETA", .num_bytes = g1_size, .derived_by_verifier = false } },
```

Comment:
I think you said ultimately the commitment to pow_zeta is not needed? Should we just be generating two challenges (zeta, alpha) from the transcript after adding `Z_PERM`?

Reply:
Yeah, definitely not needed, the polynomial is public information. Basically, I had to determine if type should be `WITNESS` `SELECTOR` or `OTHER` and I went with `WITNESS` because `OTHER` automatically triggers a commitment when generating the vk, and rather than filling with some dummy value to enable that commitment, I took this approach (maybe a bad decision). Anyway, this should all go away soon!